Scripts/ProcessIP2CountryData.swift

```diff
@@ -142,21 +142,30 @@ class Processor {
 
         /// Structure of the data should be `network,registered_country_geoname_id`
         let countryBlockPrefix: String = "Processing country blocks: "
+        var prevId: String? = nil
         lines[1...].enumerated().forEach { index, line in
             guard keepRunning else { return }
             
             let values: [String] = line
                 .trimmingCharacters(in: .whitespacesAndNewlines)
                 .components(separatedBy: ",")
             
+            let currId = values[1]
```

Comment:
This should be moved underneath the `guard` statement to avoid a possible out of range exception (the `guard` statement validates `values.count == 2`)